app/store/state/appState/data.js

```diff
@@ -1,7 +1,7 @@
 import { SET_CURRENT_USER_ID } from "../../constants/actionTypes";
 
 const initialState = {
-  currentUserID: ""
+  currentUserID: 0
```

Comment:
we should consider using convention for ID 
i suggest this:

- ids are string
- if id start with @@\ it's the id of a defect and it does't have to be in entities
ex: "@@\NotFoundEntity"  

we may want to format our ids with something like this:
`
{currentUserId: "id-123"}
`